internal/events/eventstream.go

```diff
@@ -428,14 +427,12 @@ func (a *eventStream) eventPoller() {
 					blockHeight, exists := checkpoint[sub.info.ID]
 					if !exists || blockHeight.Cmp(big.NewInt(0)) <= 0 {
 						blockHeight, err = sub.setInitialBlockHeight(ctx)
-					} else {
+					} else if !sub.inCatchupMode() {
```

Comment:
We were constantly restoring back to the checkpoint (and logging that fact) in catchup mode. No need

Reply:
Ok, this took me a bit of time to understand, so adding a note here for my future self. The checkpoint is updated in catchup mode as part of `batchComplete()` call: https://github.com/kaleido-io/firefly-ethconnect/blob/511d55a563aa7311c310ecadd4aa3040760abead/internal/events/logprocessor.go#L75-L76